src/views/Feed/Feed.js

```diff
@@ -16,18 +16,18 @@ import FeedSummary from './FeedSummary'
 import FeedUser from './FeedUser'
 
 function Feed(props) {
-  const { children, className, items, size } = props
+  const { children, className, events, size } = props
   const classes = cx('ui', className, size, 'feed')
   const rest = getUnhandledProps(Feed, props)
 
-  if (!items) {
+  if (!events) {
     return <div {...rest} className={classes}>{children}</div>
   }
 
-  const eventsJSX = items.map(({ key, event }, index) => {
+  const eventsJSX = events.map(({ key, ...eventData }, index) => {
     const finalKey = key || index
```

Comment:
Let's use some property of the `eventData` instead of the `index`.  Using `index` at all in a `key` is an anti-pattern we're moving away from.

You can see more on this in #376